README.md

```diff
@@ -1,17 +1,21 @@
 # Example repository for using GAP on Binder
 
-[![Binder](https://mybinder.org/badge.svg)](https://mybinder.org/v2/gh/gap-system/gap-docker-binder/master)
+[![Binder](https://mybinder.org/badge.svg)](https://mybinder.org/v2/gh/gap-system/gap-docker/master)
```

Comment:
Why to change from this repository to actually start another one? (of course, in this case at least it will start the container for the full installation of the latest GAP release, so that one is stable and will not be rebuilt).